src/applications/personalization/profile/components/direct-deposit/BankInfoCNPv2.jsx

```diff
@@ -247,19 +247,20 @@ export const BankInfoCNP = ({
       </p>
       <div className="vads-u-margin-bottom--2">
         <AdditionalInfo triggerText="Where can I find these numbers?">
+          {/* eslint-disable jsx-a11y/no-redundant-roles */}
           <img
             src="/img/direct-deposit-check-guide.svg"
             alt="On a personal check, find your bank’s 9-digit routing number listed along the bottom-left edge, and your account number listed beside that."
```

Comment:
Since the `alt` is repetitive with the `p` below, we can take a less verbose approach by making `alt="A personal check"` and adding `aria-describedby="check-caption"` which points to the `p` below e.g. `<p id="check-caption">`